test/profile.test.js

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import '@testing-library/jest-dom';
+import Profile from '../src/components/profile/Profile';
+import { render, fireEvent, screen } from './jest.setup';
+import { Provider } from 'react-redux';
+
+import store from '../src/redux/store'
+import { act } from 'react-test-renderer';
+
+const profile={
+  gender: "male",
+  language: "Kinyarwanda",
+  currency: "Local",
+  location: "kigali",
+  departement: "none",
+  manager:"true",
+  birthdate: "02/02/2020",
+};
+
+describe("User profile",()=>{
+  test('it renders user profile page', async() => {
+    render(<Profile />);
+    const formElement = screen.getByText('Gender');
+    expect(formElement).toBeInTheDocument();
+  });
+
+  it('It should update the user profile',()=>{
+    act(async()=>{
+      render( <Provider store={store}> <Profile /> </Provider>);
```

Comment:
Why are you wrapping this in provider when the render method is preconfigured with redux. Check the `jest.setup.js`. We created it to avoid this.